infrastructure/test_config.tf

```diff
@@ -20,3 +20,22 @@ resource "azurerm_key_vault_secret" "idam-client-secret-for-tests" {
   value     = "${data.vault_generic_secret.test_idam_client_secret.data["value"]}"
   vault_uri = "${module.key-vault.key_vault_uri}"
 }
+
+#region IdAM test user's password
+
+resource "random_string" "idam_password" {
+  length = 16
+  special = true
+  override_special = "/@\" "
```

Comment:
could you please give more detail on the usage of `override_special` as this is the first I have encountered this :)

Reply:
That was overriding the default list of special characters to use in the password. Wasn't necessary (took it from an example). Removed.